viewer/src/main/java/nl/b3p/viewer/stripes/UserLayerActionBean.java

```diff
@@ -134,6 +136,34 @@ public void stream(HttpServletResponse response) throws Exception {
         };
     }
 
+    public Resolution validate() throws FilterToSQLException, CQLException {
```

Comment:
dit moet dan in de `add()` komen of in een aparte resolution of laten we het REST design nu helemaal los?...

Reply:
Het was voor nu een methode om dit te kunnen testen.
In de toekomst hebben we een validatie backend nodig, maar dat is voor latere zorg.